source/kraken/maintenance_worker.cpp

```diff
@@ -60,6 +60,41 @@ void MaintenanceWorker::load(){
         auto data = data_manager.get_data();
         data->meta->instance_name = conf.instance_name();
     }
+
+    load_realtime();
+}
+
+
+void MaintenanceWorker::load_realtime(){
+    LOG4CPLUS_INFO(logger, "loading realtime data");
+
+    std::string queue_name = channel->DeclareQueue("", false, false, true, true);
```

Comment:
hmmm, the queue is created, but i don't see it's bound to any exchange with any routing key, is that normal?